eb_from_pr_upload_jsc-zen2.sh

```diff
@@ -41,6 +41,10 @@ if [ $EB_REPO == "easybuild-easyblocks" ]; then
     repo_pr_arg='--include-easyblocks-from-pr'
 fi
 
+if [[ $EB_PREFIX == *"5.0.x"* ]]; then
+  export EASYBUILD_FAIL_ON_MOD_FILES_GCCCORE=1
```

Comment:
@SebastianAchilles We should move this to `eb_from_pr_upload_jsc-zen3.sh`, and use `EB_BRANCH` I guess (cfr. #33)

@branfosj Can you look into changing that?

Reply:
done